assets/sample-orchestration-request.json

```diff
@@ -0,0 +1,24 @@
+{
+  "xml_message": "",
+  "xml_message_type": "ecr",
+  "configurations": {
+    "validation": {
+      "include_error_types": "error"
+    },
+    "convert_to_fhir": {
+      "root_template": "EICR",
+      "message_type": "ecr"
+    },
+    "standardization_and_geocoding": {
+      "geocode_method": "smarty",
+      "overwrite": "true",
+      "dob_format": ""
+    },
+    "message_parser": {
+      "message_format": "fhir",
+      "parsing_schema": "",
```

Comment:
I don't completely remember how the message parser works 😅 If `parsing_schema` is left empty, will it default to the one we use in LAC?

Reply:
Gotcha! That's always something we can edit later too, as we work to add functionality to the service.